### ISSUE TITLE
Selecting a language from list results in different view than using select button

### ISSUE THREAD

**Issue by jeremypw** [elementary/initial-setup#115]
## What Happened

When I selected "English..." from the list of languages by clicking on that entry the view changed to a list of language variants (Australia, Canada, UK, United States).  However if the "Select" button was pressed when "English..." was the selected entry, then the view skipped choice of idiom and went straight to the "Select Keyboard Layout" view.  This was unexpected.

## Expected Behavior

The path through the dialogs should not depend on how you activate the selected item

## Steps to Reproduce

1. Run `io.elementary.initial-setup` in Terminal
2. Click on the "English..." list entry
3. A list of variants is shown (with a back button at the top)
3. Press the back button to return to the language list
4. Make sure "English..." is the selected entry
5. Press the "Select" button
6. The "Select Keyboard  Layout" view is shown (with a back button at the bottom)

## Platform Information

Elementary 6 daily

**Comment by danrabbit** [comment]
Yeah it looks like we're not setting the sensitivity of the "Select" button correctly in these views
